vcbuild.bat

```diff
@@ -88,7 +88,7 @@ if "%i18n_arg%"=="intl-none" set i18n_arg=--with-intl=none
 @rem Skip project generation if requested.
 if defined noprojgen goto msbuild
 
-if defined NIGHTLY set TAG=nightly-%NIGHTLY%
+call :getnodeversion || exit /b 1
```

Comment:
It's legal to invoke `vcbuild noprojgen msi`. So the call to :getnodeversion should be before :project-gen.